src/app/components/map/map.component.scss

```diff
@@ -1,17 +1,17 @@
 @import "../../../styles.scss";
 
-div {
-    @include centered-style;
-    #map{
-        height: 400px; 
-        width: 95%; 
-        border-radius: 15px;
-        box-shadow: var(--box-shadow)
-    }
-  }
+span{
+    display: flex;
+    justify-content: space-between;
+    margin: 20px;
+}
+
+.buttons-div{
```

Comment:
Usually you can use the word container, it's generally more used for this type of thing